roles/jd-client/src/lib/mod.rs

```diff
@@ -57,7 +57,7 @@ pub static IS_NEW_TEMPLATE_HANDLED: AtomicBool = AtomicBool::new(true);
 pub struct JobDeclaratorClient {
     /// Configuration of the proxy server [`JobDeclaratorClient`] is connected to.
     config: ProxyConfig,
-    shutdown: Arc<Notify>,
+    pub shutdown: Arc<Notify>,
```

Comment:
Can you please add a new function to `JobDeclaratorClient::shutdown` to the first commit?